index.d.ts

```diff
@@ -889,8 +851,6 @@ declare module 'plaid' {
 
     invalidateAccessToken: AccessTokenFn<RotateAccessTokenResponse>;
 
-    deleteItem: AccessTokenFn<ItemDeleteResponse>;
```

Comment:
I dont think we rm'd this one?

Reply:
Might need to update `ItemDeleteResponse` to rm the `removed` key if we haven't done that too.